src/content/add-to-app/ios/project-setup.md

```diff
@@ -273,89 +272,95 @@ some/path/MyApp/
 ```
 
 :::warning
-Always use `Flutter.xcframework` and `App.xcframework`
-from the same directory. Mixing `.xcframework` imports
-from different directories (such as `Profile/Flutter.xcframework`
-with `Debug/App.xcframework`) causes runtime crashes.
+Always put `Flutter.xcframework` and `App.xcframework` from the same directory.
+Mixing `.xcframework` imports from different directories
+(like `Profile/Flutter.xcframework` with `Debug/App.xcframework`)
+causes runtime crashes.
 :::
 
-Link and embed the generated frameworks into your existing
-application in Xcode.  There are multiple ways to do
-this&mdash;use the method that is best for your project.
+#### Procedures
+
+You can either link or embed the generated frameworks
+into your existing app in Xcode.
+Use the method that is best for your project.
 
-#### Link on the frameworks
+##### Link on the frameworks
 
 For example, you can drag the frameworks from
-`some/path/MyApp/Flutter/Release/` in Finder
-into your target's **Build
-Settings > Build Phases > Link Binary With Libraries**.
+`/path/to/MyApp/Flutter/Release/` in the Finder into your target's
+**Build Settings > Build Phases > Link Binary With Libraries**.
```

Comment:
Can we add a screenshot for this step? For example:

![image](https://github.com/flutter/website/assets/737941/c2977d68-484b-4a30-baee-056093f80765)